gen_static.sh

```diff
@@ -0,0 +1,2 @@
+python -m pip install -r requirements.txt;
```

Comment:
Codacy found an issue: [Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6294856)